projects/src/main/java/org/odk/collect/projects/ProjectDetailsGenerator.kt

```diff
@@ -0,0 +1,23 @@
+package org.odk.collect.projects
+
+import java.net.URL
+
+object ProjectDetailsGenerator {
+    fun generateProjectDetails(urlString: String): GeneratedProjectDetails {
```

Comment:
It doesn't return `Project.New` it returns `GeneratedProjectDetails` so what do you mean?

Reply:
I mean the function could return a `Project.New` instead of a `GeneratedProjectDetails` and then we wouldn't need `GeneratedProjectDetails` at all. Just a thought but I could see there being advantages to the separation (having a different data class for the "generated" details) here.

Reply:
Oh I see. Yeah maybe it would be easier to return `Project.New`.
Fixed.